src/patient/PatientAlert.jsx

```diff
@@ -16,9 +16,9 @@ const formatEnrollmentText = (text) => {
   return formattedText;
 }
 
-const getAlertStatus = (alertName) => {
-  if (alertName.includes('critical-result')) return 'critical';
-  if (alertName.includes('abnormal-result')) return 'abnormal';
+const getAlertStatus = (alertCategories) => {
+  if (alertCategories.includes('critical')) return 'critical';
+  if (alertCategories.includes('abnormal')) return 'abnormal';
```

Comment:
Thanks!  But I think this should still be "critical-result" and "abnormal-result"?

Reply:
fyi @jomadoye  ^^